UKHomeOffice/nmsw-ui
# Package updates

Latest package updates from Dependabot

[Bump @babel/preset-env from 7.22.20 to 7.23.2](https://github.com/UKHomeOffice/nmsw-ui/pull/578)
[Bump webpack from 5.88.2 to 5.89.0](https://github.com/UKHomeOffice/nmsw-ui/pull/590)
[Bump @testing-library/jest-dom from 5.17.0 to 6.1.4](https://github.com/UKHomeOffice/nmsw-ui/pull/580)
[Bump esbuild from 0.19.3 to 0.19.5](https://github.com/UKHomeOffice/nmsw-ui/pull/591) 
[Bump @faker-js/faker from 7.6.0 to 8.2.0](https://github.com/UKHomeOffice/nmsw-ui/pull/589)

----

To test
- remove node_modules
- run `npm ci`
- run `npm tests`
- run `npm build`
- run Cypress tests - they should run
